agent-install/agent-install.sh

```diff
@@ -471,15 +670,68 @@ function download_anax_release_file() {
 }
 
 # If necessary, download the cfg file from CSS.
+# The input_file_path is either AGENT_CFG_FILE or INPUT_FILE_PATH, the second 
+# parameter tells which one it is, 1 for AGENT_CFG_FILE 2 for INPUT_FILE_PATH. 
```

Comment:
It looks like you used 1 and 0 for the second parameter when you call the function, not 1 and 2

Reply:
fixed